WebIDE.py

```diff
@@ -78,5 +79,17 @@ def get_local_ip_addr():
 print('''\nTo remotely edit Pythonista files:
    On your computer open a web browser to http://{}:8080'''.format(get_local_ip_addr()))
 
-debug(True)
-run(reloader=not PYTHONISTA, host='0.0.0.0')
+if PYTHONISTA:
+    print('''\nIf you're using Safari to connect, you can simply select "Pythonista WebIDE" from the Bonjour menu (you may need to enable Bonjour in Safari's advanced preferences).\n''')
+    NSNetService = ObjCClass('NSNetService')
+    service = NSNetService.alloc().initWithDomain_type_name_port_('', '_http._tcp', 'Pythonista WebIDE 2', 8080)
+    try:
+        service.publish()
+        debug(True)
+        run(reloader=not PYTHONISTA, host='0.0.0.0')
```

Comment:
You can remove `reloader=not PYTHONISTA`, since we're already sure we are in Pythonista